packages/core/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Core styles & components",
   "main": "lib/cjs/index.js",
   "module": "lib/esm/index.js",
+  "esnext": "lib/esnext/index.js",
```

Comment:
Out of curiosity, is there already an established precedent for using `"esnext"` for this?

Reply:
no, I don't think so. would probably be good to settle on a precedent though for future packages

Reply:
I have seen a reasonable amount of code on the web that has it this way. Webpack doesn't load `esnext` by default, which is good, because using it will require babel for most people. https://webpack.js.org/configuration/resolve/#resolve-mainfields

Reply:
Best prior art, both from a top linked result from google as well as being referenced in the few other places I tried to find information: http://2ality.com/2017/04/transpiling-dependencies-babel.html

Its not "spec" but that shouldn't matter.